rundeckapp/grails-spa/packages/ui-trellis/src/library/components/plugins/tests/pluginPropVal.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { mount } from "@vue/test-utils";
+import PluginPropVal from "../pluginPropVal.vue";
+
+const createWrapper = (prop = {}, value: string | boolean = "") => {
+  return mount(PluginPropVal, {
+    props: {
+      prop: {
+        type: "String",
+        ...prop,
+      },
+      value,
+    },
+    global: {
+      mocks: {
+        $t: (msg: string) => msg,
+      },
+    },
+  });
+};
+describe("PluginPropVal.vue", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it("renders 'Yes' or 'No' for Boolean type properties", () => {
+    const wrapperTrue = createWrapper(
+      { type: "Boolean", options: { booleanTrueDisplayValue: "Yes" } },
+      "true",
+    );
+    expect(wrapperTrue.text()).toBe("Yes");
+    const wrapperFalse = createWrapper(
+      { type: "Boolean", options: { booleanFalseDisplayValue: "No" } },
+      "false",
+    );
+    expect(wrapperFalse.text()).toBe("No");
+  });
```

Comment:
blocker: Recommend to replace the "Yes" and "No" for "Add" and "Don't Add", so it's clear that if a boolean[flag]DisplayValue is passed, component will render its value instead of the default yes/no